scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -2332,4 +2332,10 @@ psa_status_t psa_driver_wrapper_mac_abort(
     }
 }
 
```

Comment:
Integration testing: there needs to be at least build in `tests/scripts/all.sh` that uses a driver `.json` file that declares one or more entropy source. We don't yet have the machinery for this: we'll need https://github.com/ARMmbed/mbedtls/pull/5396 and perhaps a follow-up of that. We can merge the current pull request as a work in progress without integration testing.